packages/components/src/FormatRelativeDateTime/FormatRelativeDateTime.stories.mdx

```diff
@@ -0,0 +1,49 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { CivilDateTime } from "@std-proposal/temporal";
+import { FormatRelativeDateTime } from ".";
+
+<Meta
+  title="Components/FormatRelativeDateTime"
+  component={FormatRelativeDateTime}
+/>
+
+# FormatRelativeDateTime
+
+A FormatRelateDateTime displays the date and time using relative wording or
+approximate values. Note. This component only represents current date and dates
+in the past. No future dating.
+
+```ts
+import { FormatRelativeDateTime } from "@jobber/components/FormatRelativeDateTime";
+```
+
+<Canvas>
+  <Story
+    name="FormatRelativeDateTime"
+    args={{ date: new CivilDateTime(2020, 2, 26, 14, 29, 39) }}
```

Comment:
My one thought is that maybe this should be generated to be a date closer to the current date to actually show the formatRelative functionality because this just displays the date because it is so far away

Reply:
makes sense to me